services/galley/src/Galley/Intra/Client.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Galley.Intra.Client
+    ( lookupClients
+    ) where
+
+import Bilge hiding (options, getHeader, statusCode)
+import Bilge.RPC
+import Galley.App
+import Galley.Intra.Util
+import Galley.Types (UserClients)
+import Data.Id
+import Network.HTTP.Types.Method
+import Network.HTTP.Types.Status
+import Network.Wai.Utilities.Error
+
+lookupClients :: [UserId] -> Galley UserClients
+lookupClients uids = do
```

Comment:
Instead of using a plain list, can we instead use an object such as [this one](https://github.com/wireapp/wire-server/blob/develop/libs/brig-types/src/Brig/Types/Intra.hs#L100)? Perhaps we can rename that to make it a bit more generic.

Reply:
the list is only used as a function parameter, not for JSON serialization. This seems to be pretty common.